docs/usage/private-modules.md

```diff
@@ -128,8 +128,11 @@ Any `hostRules` with `hostType=packagist` are also included.
 
 ### gomod
 
-If a `github.com` token is found in `hostRules`, then it is written out to local git config prior to running `go` commands.
-The command run is `git config --global url."https://${token}@github.com/".insteadOf "https://github.com/"`.
+The `GOPRIVATE` environment variable is used to decide if a package should be fetched from the `GOPROXY` or directly from the source.
```

Comment:
Right now, Renovate does not use proxy itself and fetches directly for everything.

Reply:
If I'm not mistaken `go mod -d`, which updates the `go.sum` files uses by default the official go module proxy and as renovate doesn't explicitly disable this it will use the module proxy for sum-file updates.